<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1425</width>
    <height>326</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QTabWidget::pane { /* The tab widget frame */
      border-top: 2px solid #C2C7CB;
}

  QTabWidget::tab-bar {
      left: 5px; /* move to the right by 5px */
  }

  /* Style the tab using the tab sub-control. Note that
      it reads QTabBar _not_ QTabWidget */
  QTabBar::tab {
		
	font: 11pt &quot;微软雅黑&quot;;
      background: qlineargradient(x1: 0, y1: 0, x2: 0, y2: 1,
                                  stop: 0 #E1E1E1, stop: 0.4 #DDDDDD,
                                  stop: 0.5 #D8D8D8, stop: 1.0 #D3D3D3);
      border: 2px solid #C4C4C3;
      border-bottom-color: #C2C7CB; /* same as the pane color */
      border-top-left-radius: 4px;
      border-top-right-radius: 4px;
      min-width: 100px;
      padding: 2px;
  }

  QTabBar::tab:selected, QTabBar::tab:hover {
      background: qlineargradient(x1: 0, y1: 0, x2: 0, y2: 1,
                                  stop: 0 #fafafa, stop: 0.4 #f4f4f4,
                                  stop: 0.5 #e7e7e7, stop: 1.0 #fafafa);
  }

  QTabBar::tab:selected {
      border-color: #9B9B9B;
      border-bottom-color: #C2C7CB; /* same as pane color */
  }

  QTabBar::tab:!selected {
      margin-top: 2px; /* make non-selected tabs look smaller */
  }


/*鼠标右击菜单样式*/
QMenu {
      background-color: #ABABAB; /* sets background of the menu */
      border: 1px solid black;
  }

  QMenu::item {
      /* sets background of menu item. set this to something non-transparent
          if you want menu color and menu item color to be different */
      background-color: transparent;
  }

  QMenu::item:selected { /* when user selects item using mouse or keyboard */
      background-color: #654321;
  }

/*右下角托盘样式*/
  QMenuBar {
      background-color: qlineargradient(x1:0, y1:0, x2:0, y2:1,
                                        stop:0 lightgray, stop:1 darkgray);
  }

  QMenuBar::item {
      spacing: 3px; /* spacing between menu bar items */
      padding: 1px 4px;
      background: transparent;
      border-radius: 4px;
  }

  QMenuBar::item:selected { /* when selected using mouse or keyboard */
      background: #a8a8a8;
  }

  QMenuBar::item:pressed {
      background: #888888;
  }

</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout_3">
   <property name="spacing">
    <number>2</number>
   </property>
   <property name="margin">
    <number>0</number>
   </property>
   <item>
    <widget class="QSplitter" name="splitter_2">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="handleWidth">
      <number>4</number>
     </property>
     <widget class="QGroupBox" name="groupBox">
      <property name="title">
       <string/>
      </property>
      <layout class="QHBoxLayout" name="horizontalLayout_2">
       <property name="spacing">
        <number>2</number>
       </property>
       <property name="margin">
        <number>0</number>
       </property>
       <item>
        <layout class="QVBoxLayout" name="verticalLayout">
         <property name="spacing">
          <number>2</number>
         </property>
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout">
           <property name="spacing">
            <number>2</number>
           </property>
           <item>
            <widget class="QLabel" name="label_account_id">
             <property name="text">
              <string>期货账号</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QComboBox" name="comboBox_account_id"/>
           </item>
           <item>
            <widget class="QLabel" name="label_strategy_id">
             <property name="text">
              <string>策略编号</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QComboBox" name="comboBox_strategy_id"/>
           </item>
           <item>
            <spacer name="horizontalSpacer">
             <property name="orientation">
              <enum>Qt::Horizontal</enum>
             </property>
             <property name="sizeHint" stdset="0">
              <size>
               <width>40</width>
               <height>20</height>
              </size>
             </property>
            </spacer>
           </item>
          </layout>
         </item>
         <item>
          <widget class="QSplitter" name="splitter">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="handleWidth">
            <number>4</number>
           </property>
           <widget class="QTableView" name="tableView_order">
            <property name="styleSheet">
             <string notr="true">background-color: rgb(255, 255, 245);</string>
            </property>
            <property name="selectionMode">
             <enum>QAbstractItemView::SingleSelection</enum>
            </property>
            <property name="selectionBehavior">
             <enum>QAbstractItemView::SelectRows</enum>
            </property>
           </widget>
           <widget class="QTableView" name="tableView_trade">
            <property name="styleSheet">
             <string notr="true">background-color: rgb(255, 255, 245);</string>
            </property>
            <property name="selectionMode">
             <enum>QAbstractItemView::SingleSelection</enum>
            </property>
            <property name="selectionBehavior">
             <enum>QAbstractItemView::SelectRows</enum>
            </property>
           </widget>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
     <widget class="QGroupBox" name="groupBox_order_insert">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Maximum" vsizetype="Minimum">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="minimumSize">
       <size>
        <width>325</width>
        <height>0</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>325</width>
        <height>16777215</height>
       </size>
      </property>
      <property name="autoFillBackground">
       <bool>false</bool>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(255, 255, 245);</string>
      </property>
      <property name="title">
       <string>下单板</string>
      </property>
      <widget class="QLabel" name="label_buyorsell">
       <property name="geometry">
        <rect>
         <x>8</x>
         <y>69</y>
         <width>30</width>
         <height>16</height>
        </rect>
       </property>
       <property name="text">
        <string>买卖</string>
       </property>
      </widget>
      <widget class="QLineEdit" name="lineEdit_heyue">
       <property name="geometry">
        <rect>
         <x>70</x>
         <y>21</y>
         <width>117</width>
         <height>40</height>
        </rect>
       </property>
       <property name="sizePolicy">
        <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="text">
        <string>cu1601</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_heyue">
       <property name="geometry">
        <rect>
         <x>8</x>
         <y>32</y>
         <width>30</width>
         <height>16</height>
        </rect>
       </property>
       <property name="text">
        <string>合约</string>
       </property>
      </widget>
      <widget class="QRadioButton" name="radioButton_kaicang">
       <property name="geometry">
        <rect>
         <x>70</x>
         <y>93</y>
         <width>57</width>
         <height>19</height>
        </rect>
       </property>
       <property name="text">
        <string>开仓</string>
       </property>
      </widget>
      <widget class="QRadioButton" name="radioButton_maichu">
       <property name="geometry">
        <rect>
         <x>130</x>
         <y>69</y>
         <width>57</width>
         <height>19</height>
        </rect>
       </property>
       <property name="text">
        <string>卖出</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_kaiping">
       <property name="geometry">
        <rect>
         <x>8</x>
         <y>93</y>
         <width>30</width>
         <height>16</height>
        </rect>
       </property>
       <property name="text">
        <string>开平</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_shoushu">
       <property name="geometry">
        <rect>
         <x>8</x>
         <y>117</y>
         <width>30</width>
         <height>16</height>
        </rect>
       </property>
       <property name="text">
        <string>手数</string>
       </property>
      </widget>
      <widget class="QRadioButton" name="radioButton_pingcang">
       <property name="geometry">
        <rect>
         <x>130</x>
         <y>93</y>
         <width>57</width>
         <height>19</height>
        </rect>
       </property>
       <property name="text">
        <string>平仓</string>
       </property>
      </widget>
      <widget class="QSpinBox" name="spinBox_shoushu">
       <property name="geometry">
        <rect>
         <x>70</x>
         <y>117</y>
         <width>42</width>
         <height>21</height>
        </rect>
       </property>
       <property name="sizePolicy">
        <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
      </widget>
      <widget class="QRadioButton" name="radioButton_pingjin">
       <property name="geometry">
        <rect>
         <x>192</x>
         <y>93</y>
         <width>57</width>
         <height>19</height>
        </rect>
       </property>
       <property name="text">
        <string>平今</string>
       </property>
      </widget>
      <widget class="QRadioButton" name="radioButton_mairu">
       <property name="geometry">
        <rect>
         <x>70</x>
         <y>69</y>
         <width>57</width>
         <height>19</height>
        </rect>
       </property>
       <property name="text">
        <string>买入</string>
       </property>
      </widget>
      <widget class="QDoubleSpinBox" name="doubleSpinBox_xiadanjiage">
       <property name="geometry">
        <rect>
         <x>70</x>
         <y>157</y>
         <width>90</width>
         <height>21</height>
        </rect>
       </property>
       <property name="sizePolicy">
        <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="maximum">
        <double>99999.000000000000000</double>
       </property>
       <property name="value">
        <double>12345.670000000000073</double>
       </property>
      </widget>
      <widget class="QLabel" name="label_xiadanjiage">
       <property name="geometry">
        <rect>
         <x>8</x>
         <y>157</y>
         <width>30</width>
         <height>16</height>
        </rect>
       </property>
       <property name="text">
        <string>价格</string>
       </property>
      </widget>
      <widget class="QPushButton" name="pushButton_xiadantiaocang">
       <property name="geometry">
        <rect>
         <x>192</x>
         <y>251</y>
         <width>93</width>
         <height>28</height>
        </rect>
       </property>
       <property name="text">
        <string>调仓</string>
       </property>
      </widget>
      <widget class="QPushButton" name="pushButton_xiandan">
       <property name="geometry">
        <rect>
         <x>8</x>
         <y>218</y>
         <width>181</width>
         <height>61</height>
        </rect>
       </property>
       <property name="sizePolicy">
        <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="text">
        <string>下单</string>
       </property>
      </widget>
      <widget class="QPushButton" name="pushButton_xiadanquxiao">
       <property name="geometry">
        <rect>
         <x>192</x>
         <y>218</y>
         <width>93</width>
         <height>28</height>
        </rect>
       </property>
       <property name="text">
        <string>取消</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_pankouguadansell">
       <property name="geometry">
        <rect>
         <x>192</x>
         <y>157</y>
         <width>95</width>
         <height>16</height>
        </rect>
       </property>
       <property name="text">
        <string>卖 82990 / 3</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_dietingjia">
       <property name="geometry">
        <rect>
         <x>192</x>
         <y>178</y>
         <width>55</width>
         <height>16</height>
        </rect>
       </property>
       <property name="text">
        <string>↓88020</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_zuidashoushu">
       <property name="geometry">
        <rect>
         <x>192</x>
         <y>117</y>
         <width>23</width>
         <height>16</height>
        </rect>
       </property>
       <property name="text">
        <string>≤0</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_zhangtingjia">
       <property name="geometry">
        <rect>
         <x>192</x>
         <y>137</y>
         <width>55</width>
         <height>16</height>
        </rect>
       </property>
       <property name="text">
        <string>↑88020</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_pankouguadanbuy">
       <property name="geometry">
        <rect>
         <x>192</x>
         <y>198</y>
         <width>103</width>
         <height>16</height>
        </rect>
       </property>
       <property name="text">
        <string>买 82970 / 20</string>
       </property>
      </widget>
      <widget class="QCheckBox" name="checkBox_taoli">
       <property name="geometry">
        <rect>
         <x>192</x>
         <y>21</y>
         <width>57</width>
         <height>19</height>
        </rect>
       </property>
       <property name="text">
        <string>套利</string>
       </property>
      </widget>
      <widget class="QCheckBox" name="checkBox_baozhi">
       <property name="geometry">
        <rect>
         <x>192</x>
         <y>45</y>
         <width>57</width>
         <height>19</height>
        </rect>
       </property>
       <property name="text">
        <string>保值</string>
       </property>
      </widget>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
